src/main/java/medicaldoctor/controllers/PatientProfileController.java

```diff
@@ -0,0 +1,47 @@
+package medicaldoctor.controllers;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.TextField;
+import medicaldoctor.core.AppSession;
+
+public class PatientProfileController implements Initializable{
+
+    @FXML
+    private TextField textFirstName, textLastName, ageNumber, textSex;
+
+    @FXML
+    private TextField textMedicalInsurance, textBirthdate, textPrimaryDoctory;
+
+    @FXML
+    private TextField textStreetMailingAddress, textCityMailingAddress,state_mailing,state_bill;
+
+
+    @FXML
+    private TextField zipCodeMailingAddressNumber, textStreetBillingAddress, textCityBillingAddress, zipCodeBillingAddressNumber, textDoctor, textVisitDate;
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources){
+        if(AppSession.getPatientFlag() == 1){
```

Comment:
these kinds of flags should be booleans not ints

Reply:
also you can just have if the patient selection is null, no need for multiple values to serve the same purpose

Reply:
This is when a Patient is looked up from a table view selection. The flag is raised and it loaded a patient from AppSession.getPatient()